packages/table/style.ts

```diff
@@ -15,12 +16,30 @@ export const headerCss = css`
   border-bottom: 1px solid ${black};
 `;
 
+export const headerCellCss = css`
+  text-transform: capitalize;
+  font-weight: bold;
+`;
+
 export const cellCss = css`
   box-sizing: border-box;
   font-family: ${fontFamilySansSerif};
   border-bottom: 1px solid ${greyLight};
 `;
 
+export const innerCellCss = css`
+  height: 100%;
+  box-sizing: border-box;
+  padding: ${cellPadding}px;
+`;
+
+export const cellAlignmentCss = (textAlign: string) => css`
+  text-align: ${textAlign};
+  margin-right: ${textAlign === "right"
+    ? `${pointerSize + pointerSpace + cellPadding}px`
```

Comment:
This is here because @lilysamimi specifically requested we treat the down-pointing triangle as a hanging indent. So when right-aligned, the text of the cell content should always be aligned with the text, not the sort icon